reconcile/external_resources/manager.py

```diff
@@ -88,6 +88,56 @@ class ReconcileAction(StrEnum):
     DESTROY_ERROR = "Resource status in ERROR state"
 
 
+class ReconciliationStatus:
+    needs_secret_sync: bool = False
```

Comment:
`needs_secret_sync` could also be a `@property`

Reply:
Added as a property